dockers/docker-fpm-quagga/start.sh

```diff
@@ -2,9 +2,9 @@
 
 mkdir -p /etc/quagga
 if [ -f /etc/sonic/bgp_admin.yml ]; then
-    sonic-cfggen -m /etc/sonic/minigraph.xml -y /etc/sonic/bgp_admin.yml -t /usr/share/sonic/templates/bgpd.conf.j2 > /etc/quagga/bgpd.conf
+    sonic-cfggen -m /etc/sonic/minigraph.xml -y /etc/sonic/bgp_admin.yml -y /etc/sonic/deploymentId_asn_map.yml -t /usr/share/sonic/templates/bgpd.conf.j2 > /etc/quagga/bgpd.conf
```

Comment:
Do we allow the case that deploymentId_asn_map.yml does not exist? `-y` option will fail when the specified file does not exist so we'll need to do manual check beforehand.

I plan to add a `-Y` option into sonic-cfggen that silently ignore it when yml file does not exist, but that's not happening yet.